plugins/action_update_review_assignments_status/src/updateReviewAssignmentsStatus.ts

```diff
@@ -35,7 +35,7 @@ async function updateReviewAssignmentsStatus({
 
       return otherSelfAssignments.map(({ id, assignedSections }) => ({
         id,
-        isLocked: isLocked || assignedSections.length > 0,
+        isLocked: isLocked ? assignedSections.length === 0 : false,
```

Comment:
Do we still need `setOtherSelfAssignmentsLocked` at all? 

With the new database function to ensure consistency of assignments, there shouldn't be any other self-assignments (or any assignments at all) for the same section/level.

Come to think of it -- do we even need the "is_locked" field for review assignments at all? What does it mean for an assignment to be "locked"? They'll be unassigned if someone else takes it, not sure why we need to keep "is_locked" any more?

Maybe have a chat about this in the morn?